### PR TITLE
Bug 1954773: update the cno to use the egressfirewall flag

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
ovn-kubernetes has changed to use a flag to enable egressFirewall, the cno needs to use that flag

### CODE REVIEW

**Review by alexanderConstantinescu — APPROVED** [review]
/lgtm 

but also

/hold 

until https://github.com/openshift/ovn-kubernetes/pull/541 is in

**Comment by dcbw** [comment]
@alexanderConstantinescu don't we want to merge this first, and *then* the ovnkube bits? becuase otherwise ovnkube won't have egress firewall enabled until the CNO PR merges, and presumably we test that in CI and thus it would break?

**Comment by alexanderConstantinescu** [comment]
> @alexanderConstantinescu don't we want to merge this first, and _then_ the ovnkube bits? becuase otherwise ovnkube won't have egress firewall enabled until the CNO PR merges, and presumably we test that in CI and thus it would break?

We don't test egress firewall in CI. So in any case we have nothing to worry about :) 

The reason I put a hold on it was because I thought ovnkube errors if you provide it a flag it doesn't recognize (my memory might not serve me right on this one though...), which would then make it CrashLoop and block CI.

**Comment by sdodson** [comment]
e2e-aws-ovn-windows should no longer be run
/refresh
/skip

**Comment by squeed** [comment]
/override ci/prow/e2e-agnostic-upgrade
doesn't test ovn

**Pull request merged**
